install/envs/mac.yml

```diff
@@ -3,9 +3,10 @@ name: donkey
 channels:
   - defaults
   - conda-forge
+  - pytorch
 
 dependencies:
-  - python=3.7
+  - python=3.8
```

Comment:
Do we need to move to 3.8?

Reply:
I had some issues installing with 3.7, but I'll retry and see if those are still present

Reply:
3.7 works. I updated the enviroment configs